lib/rubocop/cop/rspec/predicate_matcher.rb

```diff
@@ -174,7 +176,7 @@ def check_explicit(node) # rubocop:disable Metrics/MethodLength
 
         def predicate_matcher_name?(name)
           name = name.to_s
-          name.start_with?('be_', 'have_') &&
+          (name.start_with?('be_', 'have_') || name == 'exist') &&
```

Comment:
maybe just add `exist` in the BUILD_IN_MATCHERS ?

Reply:
how about:
```ruby
def predicate_matcher_name?(name)
  name = name.to_s
  return false if BUILT_IN_MATCHERS.include?(name)
  name.start_with?('be_', 'have_') && !name.end_with?('?')
end
```

Reply:
Yep, that'll work.

Reply:
Oh, I've been mistaken. It will not work, as it have to return `true` for exist. Seems that we'll need another array for the build-in matchers that should not be ignored.

I see you just removed it, so with `explicit`  it will not require replacing `to exist` with `.exist?).to be_truthy`. I'm fine with leaving this out of scope and just fix the `inflect` style to use `exist` and not `be_exist`. We will have to rework the cop at some point in the future.

@bquorning WDYT?

Reply:
On second thought, `to exist` can't be auto-corrected, as it could not know if the object responds to `exist?` or `exists?`, so it's even better now that `exist` is accepted.